source/_lovelace/entities.markdown

```diff
@@ -12,41 +12,35 @@ footer: true
 
 Entities will be the most common type of card that will also be the most familiar to people using the standard interface. It groups items together very close to how groups used to do.
 
-## {% linkable_title Options %}
-
 {% configuration %}
 type:
   required: true
   description: entities
   type: string
 entities:
   required: true
-  description: "Entity id's or an `entity` object (see structure below)."
+  description: "A list of entity IDs or an `entity` object."
   type: list
+  keys:
+    entity:
+      required: true
+      description: "An `entity_id` to use in the frontend."
+      type: string
+    name:
+      required: true
+      description: "A new name for the `entity_id`."
```

Comment:
Not sure if you want to fix this in this PR, but keys like `name` or `tap_action` are optional.

Also the description text for `name` is inaccurate